packages/vaadin-login/src/vaadin-login-overlay.js

```diff
@@ -64,6 +64,7 @@ class LoginOverlayElement extends LoginMixin(ElementMixin(ThemableMixin(PolymerE
           action="{{action}}"
           disabled="{{disabled}}"
           error="{{error}}"
+          no-autofocus="{{noAutofocus}}"
```

Comment:
Two-way binding shouldn't be necessary
```suggestion
          no-autofocus="[[noAutofocus]]"
```